app/models/herbarium.rb

```diff
@@ -55,6 +58,148 @@ class Herbarium < AbstractModel
   # Used by create/edit form.
   attr_accessor :place_name, :personal, :personal_user_name
 
+  # Herbaria whose collections are searchable via MyCoPortal.
+  # https://www.mycoportal.org/portal/collections/index.php
+  # rubocop:disable Layout/LineLength
+  MCP_COLLECTIONS = {
```

Comment:
Seems like this should be a Set rather than a Hash since you aren't using the value for anything.  I also wonder if this shouldn't be in the database somewhere since I expect this list to change over time and as written this would require a code change to add or remove entries.

Reply:
Thanks for the review and comment!
Good point about putting that hash into the db. I agree it's the way to go. (I had thought about that before.)
But I want to do it as a separate PR. 
It might be possible to populate/update it (semi?) automatically with the collection numbers, depending on what I learn from Scott Bates. (See emails of today.) I need to think more about all this.